### ISSUE TITLE
Unable to display code if language is not supported

### ISSUE THREAD

**Issue by lawrenceching** [deckgo/gatsby-remark-highlight-code#27]
![image](https://user-images.githubusercontent.com/12114025/100631264-7d7da880-3366-11eb-8ab1-051a2a81c844.png)

```
Uncaught (in promise) TypeError: Cannot read property 'require' of undefined
    at DeckdeckgoHighlightCode.initLanguagesToLoad (:8000/0.js:2214)
    at DeckdeckgoHighlightCode.loadLanguages (:8000/0.js:2205)
    at DeckdeckgoHighlightCode.componentDidLoad (:8000/0.js:2146)
```

Below is the result for rendering Markdown text:
```
```text
Blablabla
\```
```
![image](https://user-images.githubusercontent.com/12114025/100631551-cf263300-3366-11eb-9186-d119345d9b56.png)

Maybe we can fallback to default language when we get an unsupported language?

**Comment by peterpeterparker** [comment]
Good idea 👍  If you are in to provide a PR, happy  to merge it :)

The source code of  the  webcomponents is available in  our main-mono-repo:

https://github.com/deckgo/deckdeckgo/tree/master/webcomponents/highlight-code

**Comment by peterpeterparker** [comment]
Funny thing, I have this issue in one of my personal blog post: https://daviddalbusco.com/blog/introducing-a-new-web-component-to-drag-resize-and-rotate

**Comment by peterpeterparker** [comment]
I just released version v2.1.1 of this plugin which inherits improvements of the peer dependency.

The component has been fixed to support more languages, theses defined by prism.js as "alias" (thx  @boradakash for the PR 👍)

In addition, there is also a small fix which can lead to the error displayed in OP's stacktrace.

Finally, instead of a fallback, if the provided language really doesn't exist, a clear msg will be displayed in the debugger. 
Maybe not the original behavior / solution discussed but after thinking about it, I thought it was a fair solution.
